assets/src/components/map.tsx

```diff
@@ -82,66 +80,6 @@ interface RecenterControlProps extends ControlOptions {
   recenter: () => void
 }
 
-interface StreetViewControlProps extends ControlOptions {
-  streetViewEnabled: boolean
-  setStreetViewEnabled: React.Dispatch<React.SetStateAction<boolean>>
-}
-
-const StreetViewControl = ({
-  streetViewEnabled: streetViewEnabled,
-  setStreetViewEnabled: setStreetViewEnabled,
-}: StreetViewControlProps): JSX.Element | null => {
-  const map = useMap()
-  const portalParent = map
-    .getContainer()
-    .querySelector(".leaflet-control-container")
-  const [portalElement, setPortalElement] = useState<HTMLElement | null>(null)
```

Comment:
I just wanted to know if it'd be a good pattern to break it out into it's own file. I can move other controls to their own files in another PR if @mbta/skate-developers are interested.

Reply:
Moving it out is probably a good call given how much code it is.

Reply:
I am in favor of breaking it out too! If we want to consider standardizing to only one (or one stateful) component per file, I've come across [this eslint rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md) we could consider adding

Reply:
Oooh I like the idea of using that rule!
I worry that that would have a large impact on our codebase but it'd probably be positive in the long run!